.nvmrc

```diff
@@ -0,0 +1 @@
+node
```

Comment:
This config file is used by `nvm`; it tells `nvm` to install the latest stable `nodejs` version available